### DISCUSSION TITLE
This worked on "iFlytec T1 4096"!

### DISCUSSION THREAD

**Discussion by larson9999** [slotThe/RemarkableLamyEraser#8] [General]
Just wanted to let you know this worked on a 'iFlytec T1 4096' I bought from ebay for $25 usd delivered for a remarkable 1. The eraser works but only if i ssh in and start i manually.

**Comment by larson9999** [comment]
Update. I believe this was uninstalled due to me having autoupdate on. I turned that off and reinstalled. Seems to be working as expected now.

Here is the link for the one I bought.https://www.ebay.com/itm/235105261524

**Comment by slotThe** [comment]
I added this in a64e7137fe082ac1f8b2d003e350d8ced3f770f6; thanks!
